lib/ansible/modules/cloud/vmware/vmware_host.py

```diff
@@ -24,7 +24,6 @@
 author:
 - Joseph Callen (@jcpowermac)
 - Russell Teague (@mtnbikenc)
-- Maxime de Roucy (@tchernomax)
```

Comment:
If you don't want notifications for VMware modules then you can add your name to `ignored` list [here](https://github.com/ansible/ansible/blob/52d8f9c3973a7ad4c8cbc077ba0733e93b4cae22/.github/BOTMETA.yml#L163) 
Bot will ignore your name for VMware issues and PRs. This will retain your author name in module.

Reply:
@Akasurde thanks for the tips.
done